assets/src/edit-story/components/inspector/inspectorProvider.js

```diff
@@ -92,10 +94,13 @@ function InspectorProvider({ children }) {
 
   useEffect(() => {
     tabRef.current = tab;
+  }, [tab]);
```

Comment:
Could use the custom `usePrevious()` hook here for improved readability throughout.